ds_judgements_public_ui/templates/includes/beta_phase_banner.html

```diff
@@ -1,4 +1,4 @@
-{% load i18n %}
+{% load %}
```

Comment:
This tag can probably go? Is this even valid?

Reply:
Removed